easybuild/tools/filetools.py

```diff
@@ -700,9 +700,17 @@ def adjust_permissions(name, permissionBits, add=True, onlyfiles=False, onlydirs
         for root, dirs, files in os.walk(name):
             paths = []
             if not onlydirs:
-                paths += files
+                if not skip_symlinks:
```

Comment:
Makes some sense.  But after switching this, we have yet another negation inside the for loop -- which has now moved up.  Though I would suggest to keep it that way to have the part where something has to be done first.